state/sshconnrequests.go

```diff
@@ -150,3 +152,17 @@ func (st *State) WatchSSHConnRequest(unitName string) StringsWatcher {
 		},
 	})
 }
+
+// CleanupExpiredRecord removes the expired record from the collection.
+func (st *State) cleanupExpiredRecord(docId string) error {
```

Comment:
Rename this method to be more ssh connection specific since this is on the shared `State` struct.